.circleci/config.yml

```diff
@@ -15,39 +15,22 @@ jobs:
             pip install \
               docker-compose==1.12.0 \
               awscli==1.11.76
-      - restore_cache:
-          keys:
-            - v1-{{ .Branch }}
-          paths:
-            - /caches/app.tar
       - run:
-          name: Load Docker image layer cache
-          command: |
-            set +o pipefail
-            docker load -i /caches/app.tar | true
-      - run:
-          name: Build application Docker image
-          command: |
-            docker build --cache-from=app -t app .
-      - run:
-          name: Save Docker image layer cache
-          command: |
-            mkdir -p /caches
-            docker save -o /caches/app.tar app
-      - save_cache:
-          key: v1-{{ .Branch }}-{{ epoch }}
-          paths:
-            - /caches/app.tar
+          name: Docker Build
+          command:
+            docker build .
       - run:
           name: Run tests
           command: |
-            docker-compose -f ./docker-compose.test.yml up
+            docker-compose -f ./docker-compose.test.yml up #test command
```

Comment:
You don't need the comment.  It's in a 'command:' block named 'Run tests' and is running something with 'test' in the name :)

Reply:
comment has been removed